source/deployment/deployContracts.ts

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+import * as fs from 'async-file';
+import * as path from 'path';
+import * as HttpProvider from 'ethjs-provider-http';
+import * as Eth from 'ethjs-query';
+import * as EthContract from 'ethjs-contract';
+import { ContractBlockchainData } from 'contract-deployment';
+import { SolidityContractCompiler } from "../libraries/CompileSolidity";
+import { ContractDeployer } from "../libraries/ContractDeployer";
+import { RpcClient } from "../libraries/RpcClient";
+
+const CONTRACT_INPUT_DIR_PATH = path.join(__dirname, "../../source/contracts");
+const CONTRACT_OUTPUT_DIR_PATH = path.join(__dirname, "../../output/contracts");
+const COMPILED_CONTRACT_OUTPUT_FILE_NAME = "augurCore";
+const GAS_AMOUNT = 3000000;
+const DEFAULT_ETHEREUM_PORT = 8545;
```

Comment:
This should be some other port since 8545 is _likely_ to be in-use by a developer.  Consider using something like `https://www.npmjs.com/package/get-port` which will get a random unused port.  It supports promises (async/await) and you can provide it with a preferred port that will be used if available.  Then if someone already has something listening on 8545 a different port will be used.

Reply:
Yeah, good point.  I'll try using `get-port` instead.